### PR TITLE
feat: add signs for new B line stations

### PR DESCRIPTION
**Pull request by digitalcora**
**Asana:** [✅ realtime-signs changes for GL-B station consolidation](https://app.asana.com/0/584764604969369/1201236005519217)

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes (compare on Splunk: [staging](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Drealtime-signs-dev%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840107.3874236) vs. [prod](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3Drealtime-signs-prod%20%22%5Berror%5D%22%20OR%20%22%5Bwarn%5D%22%20OR%20%22CRASH%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&earliest=-4h%40m&latest=now&sid=1545840137.3874305))

### CODE REVIEW

File: priv/signs.json [hunk 1]
```diff
@@ -5114,6 +5114,98 @@
       ]
     ]
   },
+  {
+    "id": "babcock_st_eastbound",
+    "headway_group": "green_b",
```
Comment (bklebe) [inline priv/signs.json:5119]: Are these the only B branch signs right now?

Comment (digitalcora) [reply priv/signs.json:5119]: Yep. At least, they are the only signs on "just" the B branch, as opposed to parts shared with other branches.

**Review by bklebe — APPROVED** [review]

File: priv/signs.json [hunk 2]
```diff
@@ -5114,6 +5114,98 @@
...
+    "pa_ess_loc": "GBAB",
+    "read_loop_offset": 30,
+    "text_zone": "e",
+    "audio_zones": ["e"],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "170136",
+          "direction_id": 0,
```
Comment (losvedir) [inline priv/signs.json:5129]:
```suggestion
          "direction_id": 1,
```

File: priv/signs.json [hunk 3]
```diff
@@ -5114,6 +5114,98 @@
...
+    "pa_ess_loc": "GAMO",
+    "read_loop_offset": 30,
+    "text_zone": "e",
+    "audio_zones": ["e"],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "170140",
+          "direction_id": 0,
```
Comment (losvedir) [inline priv/signs.json:5175]:
```suggestion
          "direction_id": 1,
```

**Review by losvedir — CHANGES_REQUESTED** [review]

**Pull request merged**